prodsys/simulation/state.py

```diff
@@ -368,10 +368,14 @@ def prepare_for_run(self):
     def activate_state(self):
         self.active = events.Event(self.env).succeed()
 
-    def process_state(self, target: List[float]) -> Generator:
+    def process_state(self, target: List[float], bol: bool) -> Generator:
         self.done_in = self.time_model.get_next_time(
             origin=self.resource.get_location(), target=target
         )
+        if bol == True:
```

Comment:
naming of bol -> give it a more descriptive name

Reply:
Commit 31